### DISCUSSION TITLE
Click birdseye panel to switch to live view

### DISCUSSION THREAD

**Discussion by thinkloop** [blakeblackshear/frigate#4911] [Ideas]
I've seen this brought up before, but now that we have sound, it would be especially nice if we were able to click or tap on an interesting active birdseye panel to temporarily bring up the hi-res live view with sound.

**Comment by jurasjo** [comment]
Yes, it would be really appreciated. For now when in full-screen/birds-eye, we have to press ESC button, then click Cameras, then select the camera we want to view full-screen.
